autotest/IntegTester/ps/www/control.tsx

```diff
@@ -182,6 +182,20 @@ function TestControl(p: ControlApi<MyConfig>) {
       >
         Execute
       </button>
+
+      <button
+        onClick={_ => {
+          const url = p.providerUrl(serviceId) + "?param1=ecs_video_test";
```

Comment:
Maybe keep the value of the test parameter generic:

```suggestion
          const url = p.providerUrl(serviceId) + "?param1=test_param_one";
```